tests/databricks/test_workflow.py

```diff
@@ -0,0 +1,144 @@
+from unittest import mock
```

Comment:
Since we don't have a CI running this yet, could you share how is the test coverage of the databricks provider ATM?

Reply:
@tatiana here is the current coverage report

```
cosmos/providers/databricks/notebook.py      72     11    85%
cosmos/providers/databricks/workflow.py      92      8    91%
```

It's a bit difficult to get the numbers up further without a tool like codecov to show specific lines but I think that we can start here and add more tests as we add more functionality. I'd also like to set up codecov for this project sooner than later.